docs/learn/documentation/versioned/index.html

```diff
@@ -83,7 +83,7 @@
   <li><a href="yarn/yarn-host-affinity.html">Host Affinity & Yarn</a></li>
   <li><a href="hdfs/producer.html">Writing to HDFS</a></li>
   <li><a href="hdfs/consumer.html">Reading from HDFS</a></li>
-  <li><a href="hdfs/yarn-security.html">Yarn Security</a></li>
```

Comment:
Thanks for the patch! I will also fix `yarn/yarn-security.md` to display correctly.